daemon/api_validate.go

```diff
@@ -325,8 +315,8 @@ func forgetValidationSet(st *state.State, accountID, name string, sequence int)
 	return SyncResponse(nil)
 }
 
-func validationSetForAssert(st *state.State, accountID, name string, sequence int) (*snapasserts.ValidationSets, error) {
-	as, err := validationSetAssertFromDb(st, accountID, name, sequence)
+func validationSetForTracked(st *state.State, tr *assertstate.ValidationSetTracking) (*snapasserts.ValidationSets, error) {
```

Comment:
I think this should s/Set/Sets/ to match what is returned,  looking at it again also maybe s/Tracked/Tracking/  ?